spec-abstract-noclosure/MC_ffg.tla

```diff
@@ -0,0 +1,101 @@
+----------------------------- MODULE MC_ffg -----------------------------
+
+EXTENDS FiniteSets
+
+\* @type: Int;
+MAX_BLOCK_SLOT == 5
+\* @type: Int;
+MAX_BLOCK_BODY == 3
+
+\* @type: Set(Str);
+VALIDATORS == {"V1", "V2", "V3", "V4"}
+
+N == 4
+
+VARIABLES
+    \* the set of all_blocks
+    \* @type: Set($block);
+    all_blocks,
+    \* the set of blocks on chain 1
+    \* @type: Set($block);
+    chain1,
+    \* the latest block on chain 1
+    \* @type: $block;
+    chain1_tip,
+    \* the set of blocks on chain 2
+    \* @type: Set($block);
+    chain2,
+    \* the latest block on chain 2
+    \* @type: $block;
+    chain2_tip,
+    \* If chain2_fork_block_number is not equal to 0,
```

Comment:
If chain1 and chain2 share only a genesis block, is chain2_for_block_number set to 0?

Reply:
if `chain2` forks right after genesis, then `chain2_for_block_number = 1`.